ding_nodelist.module

```diff
@@ -1767,7 +1767,7 @@ function ding_nodelist_preprocess(&$variables, $hook) {
 
       $links = array();
       foreach ($variables['links'] as $link) {
-        $links[] = l(t('@morelink', array('@morelink' => $link['text'])), $link['links']);
+        $links[] = l(t($link['text']), $link['links']);
```

Comment:
Basically this violates the most important thing about `t`:
> You should never use t() to translate variables...

But I'm not sure whether this could be safely replaced to a more sane implementation.